standard-cluster-services/src/main/java/org/terracotta/server/Server.java

```diff
@@ -183,8 +183,22 @@
    * @return
    */
   Properties getCurrentChannelProperties();
-
+  /**
+   * log messages to the console of the server
+   * @param message message to send
+   * @param sub additional parameters
+   */
   void console(String message, Object...sub);
-
+  /**
+   * log messages to the warning of the server
+   * @param warning message to send
+   * @param event additional parameters
+   */
   void warn(String warning, Object...event);
+  /**
+   * log messages to the auditing of the server
+   * @param message message to send
+   * @param event additional parameters
+   */
+  void audit(String message, Properties additionalIdInfo);
```

Comment:
Curious to know how the usage of this will look like. Would a raise a PR in tc-core to demonstrate the usage?

Reply:
this roughly translates to `Guardian.validate(AUDIT_OP,additionalIdInfo)` with some additions